### PR TITLE
 8340164: Open source few Component tests - Set1

### PR DESCRIPTION
**Pull request by DamonGuy**
Following tests are added as part of this PR:

/java/awt/LightweightComponent/LightweightFontTest/LightweightFontTest.java (Manual with PassFailJFrame)
/java/awt/LightweightComponent/LightWeightTabFocus/LightWeightTabFocus.java (Automated)
/java/awt/LightweightComponent/LWParentMovedTest/LWParentMovedTest.java (Automated)

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8340164](https://bugs.openjdk.org/browse/JDK-8340164): Open source few Component tests - Set1 (**Bug** - P4)

### Reviewers
 * [Prasanta Sadhukhan](https://openjdk.org/census#psadhukhan) (@prsadhuk - **Reviewer**)
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**) Review applies to [a43286f2](https://git.openjdk.org/jdk/pull/21252/files/a43286f254af82de5914b43a6d082865db068fc9)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21252/head:pull/21252` \
`$ git checkout pull/21252`

Update a local copy of the PR: \
`$ git checkout pull/21252` \
`$ git pull https://git.openjdk.org/jdk.git pull/21252/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21252`

View PR using the GUI difftool: \
`$ git pr show -t 21252`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21252.diff">https://git.openjdk.org/jdk/pull/21252.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21252#issuecomment-2381671159)

### CODE REVIEW

File: test/jdk/java/awt/LightweightComponent/LWParentMovedTest/LWParentMovedTest.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 4147246
+ * @summary Simple check for peer != null in Component.componentMoved
+ * (Component.java)
```
Comment (prsadhuk) [inline test/jdk/java/awt/LightweightComponent/LWParentMovedTest/LWParentMovedTest.java:28]: Component.java can be removed..guess it was Native*Helper inner class which made it reference Component.java but now that class is not there so reference to java file can be removed..

**Review by prsadhuk — APPROVED** [review]

**Review by prrace — APPROVED** [review]

**Comment by DamonGuy** [comment]
@prrace @prsadhuk Can I get a re-review? Forgot to add the frame return here. Added it again.

File: test/jdk/java/awt/LightweightComponent/LWParentMovedTest/LWParentMovedTest.java (hunk 1 above)
Comment (DamonGuy) [reply test/jdk/java/awt/LightweightComponent/LWParentMovedTest/LWParentMovedTest.java]: Removed

File: test/jdk/java/awt/LightweightComponent/LightWeightTabFocus/LightWeightTabFocus.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            robot.delay(100);
+            robot.keyRelease(KeyEvent.VK_TAB);
+            robot.keyRelease(KeyEvent.VK_SHIFT);
+            if (!btn2.hasFocus()) {
+                new RuntimeException("Second shift+tab failed");
+            }
+
+        } catch (Exception e) {
+            e.printStackTrace();
+        } finally {
```
Comment (prsadhuk) [inline test/jdk/java/awt/LightweightComponent/LightWeightTabFocus/LightWeightTabFocus.java:99]: redundant catch block...guess you mentioned muscle memory and will be removed in open...but seems like it's forgotten..

**Review by prsadhuk — APPROVED** [review]
But anyways approving

**Pull request closed without merging**